scripts/upload.sh

```diff
@@ -44,5 +44,11 @@ yq '.Resources.* | select(has("Type") and .Type == "AWS::Serverless::LayerVersio
 echo "Zipping the CloudFormation template"
 zip template.zip cf-template.yaml
 
+if [ -n "$VERSION_NUMBER" ]; then
+    METADATA_ARGS="repository=$GITHUB_REPOSITORY,commitsha=$GITHUB_SHA,committag=$GIT_TAG,commitmessage=$COMMIT_MSG,mergetime=$MERGE_TIME,skipcanary=$SKIP_CANARY_DEPLOYMENT,commitauthor='$GITHUB_ACTOR',release=$VERSION_NUMBER,codepipeline-artifact-revision-summary=$VERSION_NUMBER"
```

Comment:
```suggestion
    METADATA_ARGS="$METADATA_ARGS,release=$VERSION_NUMBER,codepipeline-artifact-revision-summary=$VERSION_NUMBER"
```
Suggesting adding the necessary changes to the string rather than duplicating the whole thing.

Do we also need both `release` and `codepipeline-artifact-revision-summary` if they're always the same? Can we only use one instead?

Reply:
They were both an ask from a support request a while ago, so removing might break things for that team